panel/models/tabulator.ts

```diff
@@ -478,19 +482,14 @@ export class DataTabulatorView extends HTMLBoxView {
 
   tableBuilt(): void {
     this._building = false
-    this.setHidden()
     this.setSelection()
-
-    // For remote pagination initialize on tableBuilt
-    this.setMaxPage()
-    this.tabulator.setPage(this.model.page)
-    this.setStyles()
     this.renderChildren()
-    this.setGroupBy()
```

Comment:
Why is this `this.setGroupBy` removed?

Reply:
Same as setHidden, it's better to configure this in the configuration rather than setting it after rendering.